functions/import_csv.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const HttpStatus = require("http-status-codes");
+const { firestore } = require('firebase-admin');
+const csvParser = require("csv-parser");
+const Busboy = require("busboy");
+const { db } = require("./common/context");
+
+function importCsv(req, res) {
```

Comment:
Nit: Please `git cherry-pick 7b686ebeb4cf170cc2e3bc684517f92670d0430d` so that my authorship of this file is maintained. I think if there are merge conflicts you can "Accept yours" so you don't lose your edits either. Thanks! :)

Reply:
Sorry for the delay, and that I forgot to mention you'll need to pull my commits to your local machine. You'd do something like this:
```
git checkout master
git pull upstream master
git checkout -b ginoschanges
git remote add gino-m https://github.com/gino-m/ground-platform.git
git pull gino-m csv-upload
git checkout up_csv
git cherry-pick 7b686ebeb4cf170cc2e3bc684517f92670d0430d
```

You'll then be asked to resolve merge conflicts and can commit. HTH!

Reply:
Or you may first want to try this:
```
git checkout up_csv
git pull upstream master
git remote add gino-m https://github.com/gino-m/ground-platform.git
git fetch gino-m
git cherry-pick 7b686ebeb4cf170cc2e3bc684517f92670d0430d
```

Reply:
I've pushed my changes to master and resolved conflicts instead; please disregard this comment.